kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,225 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: panoptes-production-app
+  labels:
+    app: panoptes-production-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: panoptes-production-app
+  template:
+    metadata:
+      labels:
+        app: panoptes-production-app
+    spec:
+      containers:
+        - name: panoptes-production-app
+          image: zooniverse/panoptes:__IMAGE_TAG__
+          env:
+            - name: RAILS_ENV
+              value: production
+            - name: SECRET_KEY_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-production-env-vars
+                  key: SECRET_KEY_BASE
+            - name: ATTENTION_REDIS_URL
+              value: redis://panoptes-redis.zooniverse.org:6379/1
+            - name: AWS_REGION
+              value: us-east-1
+            - name: KINESIS_STREAM
+              value: zooniverse-production
+            - name: ZOO_STREAM_KINESIS_STREAM_NAME
+              value: zooniverse-production
+            - name: ZOO_STREAM_SOURCE
+              value: panoptes
+            - name: SUBJECTS_BUCKET
+              value: zooniverse-static
+            - name: BUCKET_PATH
+              value: panoptes-uploads.zooniverse.org
+            - name: EXPORT_BUCKET_PATH
+              value: panoptes-exports.zooniverse.org
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: AWS_ACCESS_KEY_ID
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: AWS_SECRET_ACCESS_KEY
+            - name: DUMP_CONGESTION_OPTS_INTERVAL
```

Comment:
it looks like a chunk of these ENV vars are the same as the defaults in code, do we need to specify them all if they are not changing the default behaviour?

Reply:
I assumed they were config options for a reason. If there was no reason, then no

Reply:
they are configurable through the env vars but they now have defaults. Production uses the same value as the default, staging doesn't so staging is good to keep, production env var can go until we decide we want to change it.